src/client.js

```diff
@@ -374,6 +374,9 @@ export class StreamChat {
 	 * @return {string} Returns a token
 	 */
 	createToken(userID, exp) {
+		if (this.secret == null) {
```

Comment:
isn't it better to check falsy values?

Reply:
not needed in this case (you can see that in the client constructor)